src/transport_udp.cpp

```diff
@@ -84,6 +84,7 @@ TransportStatus UDPTransport::status() const {
     } else if (!isServerMode) {
         return clientStatus;
     }
+    return clientStatus;
```

Comment:
Wasn't sure about what the correct thing to do in this case is? cc @TimEvens

Reply:
Probably can drop

```
else if (!isServerMode) {
        return clientStatus;
    }
```
in favor of having the fallback to just return clientStatus.